billchop-fe/src/components/BillsListAccordion.tsx

```diff
@@ -31,10 +31,13 @@ export default class BillsListAccordion extends React.Component<
     return bills.map((bill) => {
       return (
         <Card key={bill.Id}>
-          <Accordion.Toggle style={{ cursor: "pointer" }} as={Card.Header} eventKey={bill.Id}>
+          <Accordion.Toggle className="d-flex" style={{ cursor: "pointer" }} as={Card.Header} eventKey={bill.Id}>
```

Comment:
is this className really needed?

Reply:
It's only needed for the flex-grow-1 on the name-total wrapper. Since image is set size, if I set w-100, the name and total appear on a different line from the image, as it tries to fill the whole container. Flex grow tells the wrapper to fill all available space, which accounts for the image. I'm not sure whether there is a cleaner way to make it fill available space without writing something along the lines of width: 100% - 32px.